third_party/WebKit/LayoutTests/external/wpt/webvr/idlharness.html

```diff
@@ -225,6 +225,17 @@
   VRDisplayEventReason reason;
 };
 
+[Constructor(DOMString type, VRPlaneEventInit eventInitDict)]
```

Comment:
Out of curiousity, is this needed?

Reply:
I believe so - I don't knoe exactly why we need the VRPlaneEventInit, but it doesn't build without it and this follows the pattern of the VRDisplayEventInit.